Userland/Libraries/LibGUI/WindowManager.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, The SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibCore/Object.h>
+#include <LibGUI/Event.h>
+
+namespace GUI {
+
+class WindowManager : public Core::Object {
+    C_OBJECT(WindowManager)
+
+public:
+    static WindowManager* from_wm_id(int);
+    virtual ~WindowManager() override;
+    int wm_id() { return m_wm_id; };
```

Comment:
"Meh" on introducing new heavily abbreviated function and variable names to the code base. Those should be `window_manager_id` and `m_window_manager_id` respectively (same for other occurrences of `wm`).

Reply:
i didn't introduce it, wm and wm_id was used before i did
as for using id() instead of wm_id, Window.cpp uses window_id(), so i decided to do something alike

Reply:
(it's also much easier to search for wm_id than id)

Reply:
`id` would only be used internally though, or when using a WindowManager object (i.e. `window_manager.id()`). Everything else should be named `window_manager_id` instead.

Since we are touching most of the call sites anyways, I don't think there is much harm in giving it a proper name now.

Reply:
i wonder if that's really useful, we already know the context in every place where wm_id (or wm) is used, making it clear wm stands for window manager